### PR TITLE
GuzzleCollector :: Argument 1 is "max body size"

### PR DESCRIPTION
**Pull request by xkobal**
| Q             | A
| ------------- | ---
| Bug fix?      | [yes]
| New feature?  | [no]
| BC breaks?    | [no]
| Deprecations? | [no]
| Tests pass?   | [yes]
| License       | Apache License 2.0

### CODE REVIEW

**Comment by xkobal** [comment]
@csarrazi Tell me if you prefer that I change the order of argument, that's the other way to fix it ;-)

File: src/Resources/config/collector.xml [hunk 1]
```diff
@@ -9,6 +9,7 @@
         <service id="csa_guzzle.data_collector.history_bag" class="ArrayObject" />
 
         <service id="csa_guzzle.data_collector.guzzle" class="Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector">
+            <argument type="constant">\Csa\Bundle\GuzzleBundle\DataCollector\GuzzleCollector::MAX_BODY_SIZE</argument>
```
Comment (csarrazi) [inline src/Resources/config/collector.xml]: Maybe use an empty argument here instead, as the default value is actually defined in the configuration class.

The main problem was mainly the `addArgument` call (which actually inverts both arguments, as the second argument didn't exist at the time), which should indeed be replaced by `replaceArgument`, which you actually did.

**Comment by xkobal** [comment]
Done ;-)

**Comment by csarrazi** [comment]
Good catch!

If you could just change the `<argument type="constant">...</argument>` directive to `<argument />` (and if it passes tests after this change), I'll merge it.

Actually, I didn't want to change the order of arguments, as it would be a BC break (some people may actually use the bundle outside of Symfony, in a Silex app, for example).

**Comment by csarrazi** [comment]
Thanks @xkobal!

**Pull request merged**